content/news/2013/05/2013-05-16-api-release-kit.md

```diff
@@ -9,11 +9,6 @@ topics:
   - api
   - code
   - api
-  - Census
-  - epa
-  - FCC
-  - federal-register
-  - National Weather Service
   - NREL
   - us-department-of-labor
 ---
```

Comment:
More agencies.

Reply:
nrel in https://github.com/GSA/digitalgov.gov/pull/7140 
us-department-of-labor in https://github.com/GSA/digitalgov.gov/pull/7152